test/handlers_mozilla_auth0_test.js

```diff
@@ -19,40 +19,50 @@ suite('handlers/mozilla-auth0', function() {
     });
 
     test('user for ldap profile', function() {
-      let user = handler.userFromProfile({
+      const user_id = 'ad|Mozilla-LDAP|foo';
+      const user = handler.userFromProfile({
         email: 'foo@mozilla.com',
         email_verified: true,
+        user_id,
         identities: [{provider: 'ad', connection: 'Mozilla-LDAP'}],
       });
-      assume(user.identity).to.equal('mozilla-ldap/foo@mozilla.com');
+
+      assume(user.identity).to.equal(`mozilla-auth0/${encode(user_id)}`);
     });
 
     test('user for email profile', function() {
-      let user = handler.userFromProfile({
+      const user_id = 'email|foo';
+      const user = handler.userFromProfile({
         email: 'foo@bar.com',
         email_verified: true,
+        user_id,
         identities: [{provider: 'email', connection: 'email'}],
       });
-      assume(user.identity).to.equal('email/foo@bar.com');
+
+      assume(user.identity).to.equal(`mozilla-auth0/${encode(user_id)}`);
     });
 
     test('user for google profile', function() {
-      let user = handler.userFromProfile({
+      const user_id = 'google|foo';
+      const user = handler.userFromProfile({
         email: 'foo@bar.com',
         email_verified: true,
-        identities: [{
-          provider: 'google-oauth2',
-          connection: 'google-oauth2'}],
+        user_id,
+        identities: [{provider: 'google-oauth2', connection: 'google-oauth2'}],
       });
-      assume(user.identity).to.equal('email/foo@bar.com');
+
+      assume(user.identity).to.equal(`mozilla-auth0/${encode(user_id)}`);
     });
 
     test('user for github profile', function() {
-      let user = handler.userFromProfile({
+      const user_id = 'github|0000';
```

Comment:
it'd be great to have some tests here for user_id's for which encoding is not identity, e.g., `email|with/slashes`.